src/Servers/Kestrel/test/FunctionalTests/HttpsTests.cs

```diff
@@ -427,7 +427,7 @@ public async Task ClientAttemptingToUseUnsupportedProtocolIsLoggedAsDebug()
                 using (var sslStream = new SslStream(stream, true, (sender, certificate, chain, errors) => true))
                 {
                     // SslProtocols.Tls is TLS 1.0 which isn't supported by Kestrel by default.
-                    await Assert.ThrowsAsync<IOException>(() =>
+                    await Assert.ThrowsAnyAsync<Exception>(() =>
```

Comment:
Other changed test now working fine. But this one only gets further and fails a few lines down:
``` text
Assert.Equal() Failure
Expected: 1
Actual:   2

Stack trace
   at Microsoft.AspNetCore.Server.Kestrel.FunctionalTests.HttpsTests.ClientAttemptingToUseUnsupportedProtocolIsLoggedAsDebug() in /_/src/Servers/Kestrel/test/FunctionalTests/HttpsTests.cs:line 439
--- End of stack trace from previous location where exception was thrown ---
```

I'm nervous about changing the throwing line to `Assert.Equal(2, loggerProvider.FilterLogger.LastEventId);` because a similar test in main's src\Servers\Kestrel\test\InMemory.FunctionalTests\HttpsTests.cs uses the current assertions. Thoughts @Tratcher @halter73❔

Will try that change overnight but will await your answers.

Reply:
Weird, failed with that change -- `Actual:     1` in that case. Removed the extra commit&hellip;